.github/workflows/test_integration_adapter.yml

```diff
@@ -75,7 +75,7 @@ jobs:
           python -m venv .venv
           source .venv/bin/activate
           pip install --upgrade pip
-          pip install build wheel 'fal[${{ matrix.profile }}]'
+          pip install build wheel '../../fal[${{ matrix.profile }}]'
```

Comment:
Why don't we change the `working-directory` instead?

Reply:
Because we use the `.venv` in another step in the same working directory as this one.